pandas/tests/indexes/common.py

```diff
@@ -325,6 +372,11 @@ def test_argsort(self):
 
     def test_numpy_argsort(self):
         for k, ind in self.indices.items():
+
+            # 'mixedIndex' unorderable in Python3
+            if k in ['mixedIndex']:
```

Comment:
yes this is all fine. Though you may want to add another test specifially dealing with mixedIndex, e.g. asserting that argsort raises and such.